JobConfig/primary/IPAMuminusMichel.fcl

```diff
@@ -6,16 +6,57 @@
 #
 #include "Production/JobConfig/primary/IPAStopParticle.fcl"
 
-physics.filters.PrimaryFilter.MinimumPartMom : 30.0 // MeV/c TODO - tweak this parameter to get more DetSteps
+physics.filters.PrimaryFilter.MinimumPartMom : 1.0
 physics.producers.FindMCPrimary.PrimaryProcess : "mu2eMuonDecayAtRest"
 
-physics.producers.generate : {
-  module_type : Pileup
-  verbosity : 0
-  inputSimParticles: IPAMuminusStopResampler
-  captureProducts : []
-  decayProducts : [ @local::Pileup.dioGenTool ]
-  stoppingTargetMaterial : "IPA"
+CompositeGenerator: {
+  module_type: CompositeMaterialGenerator
+  tag: IPAMuminusStopResampler
```

Comment:
When you move this to Offline/EventGenerator/fcl/prolog.fcl, set this to have a value of @nil.  If someone forgets to override this it will produce a run-time error when someone tries to read the parameter - so it fails loudly and safely.  

My own style is to never label something just "tag" - give it a descriptive name.

Reply:
The latest push in https://github.com/Mu2e/Offline/pull/1290 exchanges `tag` for `SimParticleCollection`.

Reply:
Thanks - I see the matching change here.